packages/core-entities/src/client/configuration-client.ts

```diff
@@ -480,7 +480,7 @@ export class ConfigurationClient extends ClientRegistry<AllConfigurationQuery> {
     if (!pCash.totalSupply) throw Error('pCash total supply not found');
     const currentUnderlyingSupply = pCash.totalSupply.toUnderlying();
 
-    return { maxUnderlyingSupply, currentUnderlyingSupply };
+    return { maxUnderlyingSupply, currentUnderlyingSupply, capacityRemaining: maxUnderlyingSupply.sub(currentUnderlyingSupply) };
```

Comment:
Does your eslint put this on a separate line?

Reply:
eslint doesn't auto format in this file. It works in some directories but not others idk what's wrong with it.